packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/types/ground_overlay.dart

```diff
@@ -0,0 +1,353 @@
+    required GroundOverlayId groundOverlayId,
+    required BitmapDescriptor image,
+    required LatLng position,
+    double? width,
+    double? height,
+    Offset anchor = const Offset(0.5, 0.5),
+    double bearing = 0.0,
+    double transparency = 0.0,
+    double zIndex = 0.0,
+    bool visible = true,
+    bool clickable = true,
+    VoidCallback? onTap,
+    double? zoomLevel,
+  }) {
+    return GroundOverlay._(
+      groundOverlayId: groundOverlayId,
+      image: image,
+      position: position,
+      width: width,
+      height: height,
+      anchor: anchor,
+      bearing: bearing,
+      transparency: transparency,
+      zIndex: zIndex,
+      visible: visible,
+      clickable: clickable,
+      onTap: onTap,
+      zoomLevel: zoomLevel,
+    );
+  }
+
+  /// Uniquely identifies a [GroundOverlay].
+  final GroundOverlayId groundOverlayId;
+
+  @override
+  GroundOverlayId get mapsId => groundOverlayId;
+
+  /// A description of the bitmap used to draw the ground overlay.
+  ///
+  /// To create ground overlay from assets, use [AssetMapBitmap],
+  /// [AssetMapBitmap.create] or [BitmapDescriptor.asset].
+  ///
+  /// To create ground overlay from raw PNG data use [BytesMapBitmap]
+  /// or [BitmapDescriptor.bytes].
+  final BitmapDescriptor image;
+
+  /// Geographical location to which the anchor will be fixed and the [width] of
+  /// the overlay (in meters). The [anchor] is, by default, 50% from the top of
+  /// the image and 50% from the left of the image. This can be changed by
+  /// with the [anchor] parameter.
+  /// You can optionally provide the [height] of the overlay (in meters).
+  /// If you do not provide the [height] of the overlay, it will be
+  /// automatically calculated to preserve the proportions of the image.
+  /// If [position] is specified, [bounds] must be null.
+  final LatLng? position;
+
+  /// Width of the ground overlay (in meters).
+  final double? width;
+
+  /// Height of the ground overlay (in meters).
+  final double? height;
+
+  /// Bounds which will contain the image.
+  /// If [bounds] is specified, [position] must be null.
+  final LatLngBounds? bounds;
+
+  /// The image point is specified in normalized coordinates: An anchor of
+  /// (0.0, 0.0) means the top left corner of the image. An anchor
+  /// of (1.0, 1.0) means the bottom right corner of the image.
+  /// Defaults to (0.5, 0.5), which is the center of the image.
+  final Offset? anchor;
+
+  /// The amount that the image should be rotated in a clockwise direction. The
+  /// center of the rotation will be the image's anchor.
+  /// The default bearing is 0, i.e., the image is aligned so that up is north.
+  final double bearing;
+
+  /// The transparency of the ground overlay. The default transparency is 0 (opaque).
+  final double transparency;
+
+  /// The tile overlay's zIndex, i.e., the order in which it will be drawn where
+  /// overlays with larger values are drawn above those with lower values
```

Comment:
Missing trailing period:
```suggestion
  /// overlays with larger values are drawn above those with lower values.
```